CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## UNRELEASED
+
+### Adds
+
+* Tasks can now be registered with the `afterModuleReady` flag, which is more useful than `afterModuleInit` because it waits for the module to be more fully initialized, including all "improvements" loaded via npm. The original `afterModuleInit` flag is still supported in case someone was counting on its behavior.
```

Comment:
We need this in order for an external improvement to be able to help with the asset build process. It was an oversight all along that it didn't work this way, but I'm being careful for bc.